bundles/org.openhab.automation.jythonscripting/README.md

```diff
@@ -1,5 +1,13 @@
 # Jython Scripting
 
+::: tip Note:
+Currently, the development of Jython stopped at version 2.7 with no definite timeline to support Python 3.x.
+The 3rd party openHAB helper library for Jython is also no longer maintained.
+We would not recommend using Jython scripting at this point in time.
+For alternatives, check out [JS Scripting](../jsscripting/) and [JRuby](../jrubyscripting/).
```

Comment:
I wonder if it makes sense to call out these two or just point them to the Automation Add-ons page. I'm sensitive to privileging any of the add-ons over the others too much. The thing that makes JS Scripting and jRuby really good are their helper libraries. But Groovy and Java Scripting are built around using the native Java APIs and do so much more naturally than even Rules DSL does. So is a lack of a helper library there a disadvantage? I'm not sure. 

I also don't want to have to come back here if at some point in the future Common Lisp, Scheme, or even Clojure are added as options. Functional languages are objectively superior after all. 😜

Reply:
Should we mention HabApp here?

Reply:
I've been ambivalent on that. Since HABApp is not actually a part of the OH project I'm cautious about including it in the official docs too much. However, it makes a whole lot of sense to mention it here.

I'm thinking yes, we should point the users to HABApp.

Reply:
i think we should just mention the other supported language engines included with openHAB and not link to anything external.

Reply:
The only reason why I think it's worth mentioning is because HABApp is the *only* viable way to use Python with OH. It feels like it does a disservice to the users who are clearly wanting to use Python because they are on this page in the first place to not mention the only way they can use Python with OH. 

Ideally, with all due respect to what HABApp is and what it does, I'd like to see a GraalVM Python add-on created. But after two years there's no sign of anyone wanting to do that. 

It feels wrong to essentially say "sorry, you'll have to learn a new programming language" when there *is* a viable Python option.

Note: I usually fall on the "don't mention HABApp because it's external" side of things, this one case feels different.